docs/statistics.md

```diff
@@ -1,269 +1,425 @@
 
-## pktSndDropTotal
+### pktSndDropTotal
 
-The number of "too late to send" packets dropped by sender (refer to `TLPKTDROP`).
+The total number of "too late to send" packets dropped by the sender (refer to `SRTO_TLPKTDROP` in [API.md](API.md)). Available for sender.
+
+TODO: Update what's below -> Maxim to check -> issue on Github - update RFC and here put the link
 
 The total delay before `TLPKTDROP` is triggered consists of the `SRTO_PEERLATENCY`, 
 plus `SRTO_SNDDROPDELAY`, plus 2 * the ACK interval (default ACK interval is 10 ms).
 The delay used is the timespan between the very first packet and the latest packet 
 in the sender's buffer.
 
+### pktRcvDropTotal
+
+TODO: 
 
-## pktRcvDropTotal
+- missing?
+- Revise the text
+- create ticket -> rename back SRTO_TSBPDDELAY -> Maxim to create
 
-The number of "too late to play" missing packets. Receiver only.
+The total number of "too late to deliver" missing packets. Available for receiver.
 
-TSBPD and TLPacket drop socket option should be enabled.
+Missing packets here means lost or not yet received out-of-order packets. The receiver drops only those packets that are missing by the time there is at least one packet ready to be delivered to the upstream application.
 
-The receiver drops only those packets that are missing by the time there is at 
-least one packet ready to play.
+Also includes packets that failed to be decrypted (see [pktRcvUndecryptTotal](#pktRcvUndecryptTotal)). These 
+packets are present in the receiver's buffer and not dropped at the moment the decryption has failed. 
 
-Also includes packets that failed to be decrypted (pktRcvUndecryptTotal). These 
-packets are present in the receiver's buffer, and not dropped directly. 
+Note that `SRTO_TSBPDMODE` and `SRTO_TLPKTDROP` socket options should be enabled (refer to API.md - link!!!!).
 
-## pktRcvUndecryptTotal
+### pktRcvUndecryptTotal
 
-The number of packets that failed to be decrypted. Receiver side.
+The total number of packets that failed to be decrypted at the receiver side. Available for receiver.
 
-## pktSndFilterExtraTotal
+TODO: Is it correct that we include this number in the other statistics?
 
-The number of control packets supplied by the packet filter (refer to
+### pktSndFilterExtraTotal
+
+TODO: 
+
+- Packet filter control packets are SRT DATA packets. -> put this for all related stats
+- put this in FEC docs
+
+The total number of packet filter control packets supplied by the packet filter (refer to
 [SRT Packet Filtering & FEC](packet-filtering-and-fec.md)). Sender only.
 
-Introduced in v1.4.0.
+Packet filter control packets are SRT DATA packets.
+
+Introduced in SRT v1.4.0.
 
-## pktRcvFilterExtraTotal
+### pktRcvFilterExtraTotal
 
-The number of control packets received and not supplied back by the packet filter
+TODO: 
+
+- I do not understand the description, check the whole group.
+- for the whole group we should also mention that this statisitcs is available if FEC option is enabled
```

Comment:
This is simply a number of received FEC control packets. The description is very general because this stat is calculated in the packet filter framework code, not in the FEC code.